cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/ApplyLogic.java

```diff
@@ -38,12 +44,57 @@ public ApplyLogic(@Nonnull DefaultCloudFoundryOperations cfOperations) {
         this.cfOperations = cfOperations;
     }
 
-    //TODO update the documentation as soon as the method does more than just creating applications
     /**
-     * Creates all provided applications that are not present in the live system. In case of any error, the procedure
-     * is discontinued.
-     * @param desiredApplications the applications that should all be present in the live system after the procedure
-     * @throws ApplyException if an error occurs during the procedure
+     * Assign users as space developers that are not present in the live system and
+     * revoke space developers permission, if its present in the live system but not
+     * defined in <code>desiredSpaceDevelopers</code>. In case of any error, the
+     * procedure will be continued and the error is logged.
```

Comment:
To be consistent with the other PR #104 
```suggestion
     * defined in <code>desiredSpaceDevelopers</code>. In case of any non-recoverable error, the
     * procedure is discontinued.
```